### DISCUSSION TITLE
GitHub actions log showing empty "Error:" lines

### DISCUSSION THREAD

**Discussion by jensmaurer** [community/community#7381] [Actions] [answered]
I think I have followed https://docs.github.com/en/actions/quickstart and the action indeed runs on "git push", but when inspecting the actions log, the expandable headlines just show "Error" and not the detailed log data:

![github-actions](https://user-images.githubusercontent.com/23412755/140584114-72ef980d-a717-491a-9040-5c055905374a.png)

Link: https://github.com/jensmaurer/tmp/runs/4122286152?check_suite_focus=true

(This reproduces an issue that was originally observed in this repository: https://github.com/cplusplus/draft/actions )

On a (totally unrelated) GitHub Enterprise setup, this seems to work correctly. Any help greatly appreciated.

**Comment by ethomson** [comment]
Hi @jensmaurer!  Apologies for the inconvenience.  I'm looking at https://github.com/jensmaurer/tmp/runs/4122286152?check_suite_focus=true right now and everything looks good to me?  🤔  Possibly an intermittent bug that you ran into that we've since fixed?

Can you take another look and let me know if it looks good to you or if there's a problem?

**Comment by jensmaurer** [comment, marked as answer]
Thanks for getting back to me. It seems my web content filter got in the way.
It seems that everything works after enabling JavaScript from pipelines.actions.githubusercontent.com for XHR.
(It is seriously unhelpful to have github.com content served by different top-level domains, i.e. some not under github.com.)
